README.md

```diff
@@ -104,7 +104,11 @@ For each host, copy the nebula binary to the host, along with `config.yaml` from
 
 ## Building Nebula from source
 
-Download go and clone this repo. Change to the nebula directory.
+Download go and clone this repo. 
```

Comment:
```suggestion
[Download go](https://go.dev/doc/install) and clone this repo. 
```

maybe make it part of the sentence instead of a separate link? Then you can keep the original form 🤷